datadog_checks_downloader/datadog_checks/downloader/cli.py

```diff
@@ -64,6 +96,12 @@ def download():
     if version and not __is_canonical(version):
         raise NonCanonicalVersion(version)
 
+    if root_layout_type == 'extras':
+        shipped_integrations = __find_shipped_integrations()
+        if standard_distribution_name in shipped_integrations:
+            sys.stderr.write('WARNING: {} is a known core integration'.format(standard_distribution_name))
```

Comment:
I still wonder whether we should just warn, or also ask user to confirm this action?

Reply:
I don't know how Go would handle interactively. Also, we wouldn't want automated scripts to block.

Reply:
Good points. I hate to propose the idea, but maybe another flag which acts as confirmation in this case, otherwise bail?